crates/bevy_math/src/bounding/bounded3d.rs

```diff
@@ -189,70 +203,69 @@ impl BoundingSphere {
             sphere: Sphere { radius },
         }
     }
-
-    /// Get the radius of the bounding sphere
-    #[inline(always)]
-    pub fn radius(&self) -> f32 {
-        self.sphere.radius
-    }
 }
```

Comment:
Hmm, I feel like `radius` is something you'd expect a bounding sphere/circle to have, even if `half_size` does the same thing. I'd maybe keep this, but add a doc comment like "Equivalent to [`BoundingSphere::half_size`]."

Alternatively, we could just add a `radius` doc alias to the `half_size` methods, but I'd still maybe expect it to be an actual method... not sure